css_advanced/electives-style.css

```diff
@@ -0,0 +1,99 @@
+.all-contents {
+  display: flex;
+  flex-direction: column;
+}
+
+h2 {
+  color: #97bf0e;
+  margin: 0px;
+}
+
+.course-info {
+  font-style: italic;
+}
+
+.button-shape {
+  border: 1px solid #c3c3c2;

```

Comment:
When you click on the button there is this black border appearing around it. It should not be there.